.github/workflows/build-and-release-maven.yml

```diff
@@ -91,12 +114,24 @@ jobs:
       - uses: actions/checkout@v3
         with:
           persist-credentials: false
-      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.36.0
+      - name: "Setup Java Build with specific configuration"
+        if: inputs.java-version && inputs.java-distribution
+        uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.36.0
+        with:
+          java-version: ${{ inputs.java-version}}
+          java-distribution: ${{ inputs.java-distribution}}
+      - name: "Setup Java Build with default configuration"
+        if: inputs.java-version == '' || inputs.java-distribution == ''
+        uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.36.0
```

Comment:
Actually, I didn't initialize the `java-version` and `java-distribution` to the default values since I thought if we update the default values of these properties in the `setup-java-build` action, then we would have to update it in this file as well. But if  that is okay, I will update these to their default values in this file as well. Please share your thoughts.

Reply:
I also prefer to have a defaults set here so we can switch when we want to not when the upstream decides to roll it out.

Reply:
Okay sure. I will update to the default values.

Reply:
Agreed, I had the same thought :)